config/kibana.yml

```diff
@@ -61,6 +61,11 @@
 # must be a positive integer.
 # elasticsearch.requestTimeout: 30000
 
+# List of Kibana client-side headers to send to Elasticsearch. To send *no* client-side
+# headers, set this value to null, not []. Setting it to [] will send the default list
```

Comment:
Hmm... that is frustrating, but I do think this is a burden we must overcome programmatically rather than placing it on the user.  @spalger do you have any idea how we can make joi allow empty arrays?